components/widgets/badges/badge.jsx

```diff
@@ -23,8 +23,10 @@ export default class Badge extends React.Component {
             return null;
         }
         return (
-            <div className={'Badge ' + this.props.className}>
-                {this.props.children}
+            <div className='BadgeContainer'>
```

Comment:
Still, the root element should be the `Badge` element here, not `BadgeContainer`. Actually, the camel case is only for the component itself, any other element in the component doesn't should use camel case. I would suggest to use here `className={'Badge ' + this.props.className}` and in the div inside use something like `Badge__box`.

Reply:
Probably this change will need some other css modifications.